napari/__init__.py

```diff
@@ -43,7 +43,7 @@
 vispy_logger.setLevel(logging.WARNING)
 
 from .viewer import Viewer
-from . import keybindings
+from . import _viewer_key_bindings
```

Comment:
Why does this need to be imported here if it's not used and it's private?

Reply:
The import is needed as inside that file the Viewer object get's decorated with the keybindings. Before the method was not private, but probably shouldn't have been used. We use this pattern with our layers quite frequently. Maybe I should add a `del _viewer_key_bindings` after the import so it doesn't pollute the namespace? What do you and @kne42 think?

Reply:
I will add a comment about the import doing the decoration too. I will propagate this change into all the keybindings files

Reply:
done